ios/Video/RCTVideo.swift

```diff
@@ -1254,6 +1285,8 @@ class RCTVideo: UIView, RCTVideoPlayerViewControllerDelegate, RCTPlayerObserverH
 
             if self._videoLoadStarted {
                 all(RCTVideoUtils.getAudioTrackInfo(self._player), RCTVideoUtils.getTextTrackInfo(self._player)).then { audioTracks, textTracks in
+                    self._livestream = _playerItem.duration.isIndefinite
```

Comment:
I don't think this is the good way to check if playback is Live.
You can have streams with 30min of backward buffer, In that case, I think you will have a valid duration.
please see:
https://stackoverflow.com/questions/6666478/is-there-a-way-to-distinguish-between-a-live-stream-and-an-on-demand-file-stream